### PR TITLE
Refactor login form

### PR DESCRIPTION
**Pull request by DavidKang**
Recently, we added a new helper module regarding authentication
protocol.

### CODE REVIEW

File: src/api/app/views/webui/session/_form.html.haml [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
   .form-group
-    = label_tag(:password, 'Password')
+    = label_tag(:password, 'Password', class: "#{text_color}")
     = password_field_tag(:password, nil, placeholder: 'Password', required: true, id: 'user-password', class: 'form-control')
-  = submit_tag('Log In', name: 'login', class: 'btn btn-primary')
-  %span
-    - if proxy
-      or
-      = link_to('Register', CONFIG['proxy_auth_register_page'] + '?%22')
-    - elsif CONFIG['frontend_ldap_mode'] != :on
```
Comment (DavidKang) [inline src/api/app/views/webui/session/_form.html.haml]: `CONFIG['frontend_ldap_mode']` is still used? As a different propose than `CONFIG['ldap_on']`?

File: src/api/app/views/layouts/webui/_login_form.html.haml [hunk 2]
```diff
@@ -1,22 +1,9 @@
...
-          = hidden_field_tag(:url, request.fullpath)
-        .form-group
-          = label_tag(:username, 'Username', class: 'text-light')
-          = text_field_tag(:username, nil, placeholder: 'User Name', required: true, class: 'form-control')
-        .form-group
-          = label_tag(:password, 'Password', class: 'text-light')
-          = password_field_tag(:password, nil, placeholder: 'Password', required: true, class: 'form-control')
-        .clearfix
-          = submit_tag('Log In', class: 'btn btn-success float-right')
+      = render partial: 'webui/session/form', locals: { text_color: 'text-light', submit_position: 'float-right', with_sign_up: false }
```
Comment (saraycp) [inline src/api/app/views/layouts/webui/_login_form.html.haml]: I would suggest to rename `text_color` to `label_css`, and `submit_position` to `button_css`.

Comment (DavidKang) [reply src/api/app/views/layouts/webui/_login_form.html.haml]: Thanks!, done

File: src/api/app/views/webui/session/_form.html.haml (hunk 1 above)
Comment (vpereira) [reply src/api/app/views/webui/session/_form.html.haml]: interesting that in the documentation I just find mention to `ldap_mode` https://openbuildservice.org/help/manuals/obs-admin-guide/obs.cha.administration.html#_obs_proxy_mode_configuration

**Comment by vpereira** [comment]
@DavidKang @saraycp is that ready to go?

**Comment by DavidKang** [comment]
@vpereira, yes, only waiting for @saraycp.

**Pull request merged**